internal/config/config.go

```diff
@@ -64,6 +64,13 @@ func NewConfig(fileName FilePath) *Config {
 			L2PollInterval: getEnvInt("L2_POLL_INTERVAL"),
 		},
 
+		MetricsConfig: &metrics.Config{
+			Host:              getEnvStr("METRICS_HOST"),
+			Port:              uint64(getEnvInt("METRICS_PORT")),
```

Comment:
Let's just store it as an `int` in the actual config and convert to a 64 bit type where it's used